eq-author-api/config/knexfile.js

```diff
@@ -1,12 +1,20 @@
 module.exports = {
   development: {
     client: "postgresql",
-    connection: process.env.DB_CONNECTION_URI
+    connection: process.env.DB_CONNECTION_URI,
+    pool: {
+      min: 2,
+      max: 10
+    }
   },
 
   test: {
     client: "postgresql",
-    connection: process.env.DB_CONNECTION_URI
+    connection: process.env.DB_CONNECTION_URI,
+    pool: {
+      min: 2,
+      max: 10
+    }
```

Comment:
Can we do away with separate configurations or is one specifically required for test?
If they're the same maybe they can be combined into one config:

```js
const baseDbConfig = {
	client: "postgresql",
    connection: process.env.DB_CONNECTION_URI,
    pool: {
      min: 2,
      max: 10
    }
}

module.exports = {
	development: baseDbConfig,
    test: baseDbConfig,
    production: {
	  ...baseDbConfig,
      pool: {
        min: 2,
        max: 20
      }
   }
}

```

Reply:
👍 already done